adapters/flipp/flipp.go

```diff
@@ -0,0 +1,225 @@
+package flipp
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/buger/jsonparser"
+	"github.com/gofrs/uuid"
+	"github.com/prebid/openrtb/v17/openrtb2"
```

Comment:
We have upgraded the version of the openrtb library from v17 to v19. Please update your import statements to use
`github.com/prebid/openrtb/v19/openrtb2`